tools/cli/commands/docker.js

```diff
@@ -3,19 +3,133 @@
  */
 import { spawnSync } from 'child_process';
 import chalk from 'chalk';
+import { createWriteStream, existsSync, copyFileSync, readFileSync, writeFileSync } from 'fs';
+
+const dockerFolder = `tools/docker`;
 
 /**
- * Default handler for the monorepo Docker commands.
+ * Sets default options that are common for most of the commands
  *
- * @param {object} argv - Arguments passed.
+ * @param {Object} yargs Yargs
+ * @return {Object} Modified Yargs object
  */
-const defaultDockerCmdHandler = argv => {
-	const args = process.argv.slice( 4 ); // node pnpm run docker <cmd> [args..]
+const defaultOpts = yargs =>
+	yargs
+		.option( 'type', {
+			alias: 't',
+			describe: 'Container type',
+		} )
+		.option( 'name', {
+			alias: 'n',
+			describe: 'Project name',
+		} )
+		.option( 'port', {
+			alias: 'p',
+			describe: 'WP port',
+		} );
+
+/**
+ * Gets a project name from the passed arguments. Defaults to 'dev' if not specified.
+ *
+ * @param {Object} argv Yargs
+ * @return {string} Project name
+ */
+const getProjectName = argv => {
+	let project = 'dev';
+	if ( argv.type === 'e2e' ) {
+		project = argv.name ? argv.name : 'e2e';
+	}
+
+	return project;
+};
+
+/**
+ * Builds a map of ENV variables for specified configuration
+ *
+ * @param {Object} argv Yargs
+ * @return {Object} key-value pairs of ENV variables
+ */
+const buildEnv = argv => {
+	const envOpts = { PORT_WORDPRESS: 80 };
```

Comment:
This breaks the ability to override `PORT_WORDPRESS` via `.env` for the 'dev' environment.

Reply:
Nice catch!

Reply:
fixed in 
bda80c7